acceptancetests/deploy_stack.py

```diff
@@ -1154,3 +1228,15 @@ def wait_for_state_server_to_shutdown(host, client, instance_id, timeout=60):
         else:
             raise Exception(
                 '{} was not deleted:'.format(instance_id))
+
+
+def deploy_and_test(test, args):
```

Comment:
Minor quibble. Not sure I like the verbiage here 'deploy_and_test'. Calling this boostrap doesn't fit really, but open to other ideas if you have them.